### DISCUSSION TITLE
Exit node DGA spoofing

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [nymtech/nym#4390] [Ideas]
user ---> Exit node ---> [DGA](https://en.wikipedia.org/wiki/Domain_generation_algorithm) ---> Internet

This reduces the risk of node address exposed.

**Comment by gyrusdentatus** [comment]
> user ---> Exit node ---> [DGA](https://en.wikipedia.org/wiki/Domain_generation_algorithm) ---> Internet
> 
> This reduces the risk of node address exposed.

Wow, this sounds amazing , a novel and interesting approach to say the least imho 

!!!. Although …it might be a bit dated nowadays maybe ? 

With all the ML that is going on, novel attacks and fingerprinting on tor …? Read a bunch of crazy stuff a few months back. 

Could you expand on the idea or how it would be implemented without too much of a performance hit? 
Would the pros outweigh the cons? 

Do we even need clearnet? do we need DNS for this stuff? I mean internet is not just HTTP , right? :)
